docs/manual/introduction/Animation-system.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+    <h1>[name]</h1>
+       
+    <h2>Overview</h2>
+
+    <div class="desc">
+        Within the three.js animation system you can animate various properties of your models:
+        the bones of a [page:SkinnedMesh skinned and rigged model], 
+        [page:Geometry.morphTargets morph targets], different material properties (colors, 
+        opacity, booleans), visibility and transforms. The animated properties can be faded in,
+        faded out, crossfaded and warped. The weight and time scales of different simultaneous 
+        animations on the same object as well as on different objects can be changed 
+        independently. Various animations on the same and on different objects can be 
+        synchronized.<br /><br />
+
+        To achieve all this in one homogeneous system, the three.js animation system
+        [link:https://github.com/mrdoob/three.js/issues/6881 has completely changed in 2015]
+        (be aware of outdated informations!), and it has now an architecture similar to 
+        Unity/Unreal Engine 4. Here is a short overview over its main components and how 
+        they work together:
+        
+    </div>
+           
+    <h3>Animation Clips</h3>
+        
+    <div class="desc">        
+       
+        If you have successfully imported an animated 3D object (it doesn't matter if it has
+        bones or morph targets or both) - for example exporting it from Blender with the 
+        [link:https://github.com/mrdoob/three.js/tree/master/utils/exporters/blender/addons/io_three Blender exporter] and
+        loading it into a three.js scene using [page:JSONLoader] -, one of the geometry's 
```

Comment:
Ah – now I see, what you meant.

But I think, the dash is correct: Since the part between the two dashes is a parenthesis, deleting the second dash would create wrong references (from my point of view the part after the comma would look as if it would belong directly to “for example …” , not continuing and completing the superior sentence).

But be that as it may – I think it’s not so relevant to be worth a further PR 😉.